### ISSUE TITLE
AttributeError: 'IOAdapter' object has no attribute 'unpad'

### ISSUE THREAD

**Issue by duckduck-sys** [hmorimitsu/ptlflow#23]
Following the [colab example ](https://colab.research.google.com/drive/1YARBRUGplqTRnRuY9sKIs6LY_2kWAWZJ?usp=sharing), the [line](https://colab.research.google.com/drive/1YARBRUGplqTRnRuY9sKIs6LY_2kWAWZJ#scrollTo=XlNUCGv_aWyz&line=6&uniqifier=1
):

```
# Some padding may have been added during prepare_inputs. The line below ensures that the padding is removed
# to make the predictions have the same size as the original images.
predictions = io_adapter.unpad(predictions)
```

gives the error 

`AttributeError: 'IOAdapter' object has no attribute 'unpad'`

A simple fix seems to be to replace the line with

`predictions = io_adapter.unpad_and_unscale(predictions)`

which makes the example run. Is this correct?

**Comment by hmorimitsu** [comment]
Yes, that is correct. Thank you for reporting this error. I will update the example later.

**Comment by zplizzi** [comment]
It is still incorrect here: https://ptlflow.readthedocs.io/en/latest/starting/inference.html

**Comment by hmorimitsu** [comment]
Thank you for the update! I will push a fix soon.

**Comment by hmorimitsu** [comment]
Docs were fixed in #29 .
